src/components/Inputs/CorInputTime/InputTime.module.scss

```diff
@@ -0,0 +1,33 @@
+@import '../../../variables';
+
+.inputTime {
```

Comment:
Can we agree on using either kebab or snake case for CSS classes ?

Reply:
I'm more of the camelCase team